_sass/_home.scss

```
@@ -180,13 +180,16 @@
 
   .testimonial-grid {
     @include grid-base($grid-min-width-large, $grid-gap-large);
-    max-width: $container-medium-width;
+    max-width: $container-max-width;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)